data/human/post-war reactions.txt

```diff
@@ -11,65 +11,55 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
-########################################################################
-
-# This file contains human reactions to the pug war, and events
-# leading up to it.  It lets us know how regular, everyday people felt
-# about the war. This leads to the reveal of the Hai, who are unable
-# to stay hidden with so many people thinking about aliens.
-
-mission "Post-War Reactions Begin"
+mission "Main Plot Completed Date"
 	landing
 	invisible
 	to offer
 		has "main plot completed"
 	on offer
-		event "post-war reactions begin" 20 30
+		"main plot completed date" = "days since start"
 		fail
 
-event "post-war reactions begin"
-
 
 
 mission "War Reactions: Asteroid Miners"
-	landing
 	minor
+	landing
 	source
 		government "Republic"
-		# These miners were far from the war the whole time.
 		attributes "deep" "paradise" "north"
-		not attributes uninhabited
+		attributes "spaceport"
 	to offer
-		has "event: post-war reactions begin"
+		"days since start" - "main plot completed date" > 30
```

Comment:
This new condition will initially be 0, so this expression will retrun true as soon as `days since start` exceeds 30, which is bad.